frontend/packages/kubevirt-plugin/src/components/boot-order/boot-order-summary-empty-state.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { List, ListItem, Expandable, Text, TextVariants } from '@patternfly/react-core';
+import { BootableDeviceType } from '../../types';
+import { deviceLabel, deviceKey, bootOrderEmptyTitle, bootOrderEmptyMessage } from './constants';
+
+import './boot-order-summary.scss';
+
+export const BootOrderSummaryEmptyState = ({ devices }: BootOrderSummaryEmptyStateProps) => {
+  const [isExpanded, setIsExpanded] = React.useState<boolean>(false);
+
+  const options = devices.filter((device) => !device.value.bootOrder);
+
+  const onToggle = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  // Note(Yaacov): className='text-secondary' is a hack to fix TextVariants being overriden.
+  return (
+    <>
+      <Text component={TextVariants.p} className="kubevirt-boot-order-summary__empty-text">
+        {bootOrderEmptyTitle}
+      </Text>
+      <Text component={TextVariants.small} className="text-secondary">
+        {bootOrderEmptyMessage}
+      </Text>
+      {options.length > 0 && (
+        <Expandable
+          toggleText={isExpanded ? 'Hide default boot disks' : 'Show default boot disks'}
+          onToggle={onToggle}
+          isExpanded={isExpanded}
+          className="kubevirt-boot-order-summary__expandable"
+        >
+          <List>
```

Comment:
This should be an ordered list but I see that the list component doesn't come with the option. It would be great if we could contribute that back to PatternFly, but in the meantime what's the easiest way to ensure this is numbered? I think it makes a big difference, WDYT @Ranelim

Reply:
Added a commit that use ordered list:
![matt · Details · OKD](https://user-images.githubusercontent.com/2181522/69674700-b32de880-10a5-11ea-9354-45293cb01449.png)

p.s.
@matthewcarleton  Will open a PR / Discussion in patternfly's github and cc you.

Reply:
https://github.com/patternfly/patternfly-react/issues/3356

Reply:
> LGTM @Ranelim WDYT?
Not sure I follow what images I need to update where. The Boot-order PR and the images u added to your PR seem aligned.  
But the only difference is how the CD-ROM string appears. Adding @matthewcarleton to confirm this. The design  display:
Drive [1]: [container] (CD-COM)
And not:
CD-drive-[1] (CD-ROM)
This is fetched from the CD-COM creation.
The “Drive [#]” is the field title, and the rest is the field name itself.

Reply:
ya good catch @Ranelim I didn't notice that. I agree we want to keep the proper format for CD-ROMs here.

Reply:
I think the lack of consistency here is quite unfortunate.

Also it can be quite difficult to understand this constellation:

1. Drive 2
2. Drive 1

I think the names would be a better solution here.

Reply:
@matthewcarleton @Ranelim this PR is continuing in: https://github.com/openshift/console/pull/3436 please open/continue this discussion there.

#3436 also handle the modal that use the same device labeler, so will also benefit from this discussion.

Reply:
thanks @yaacov see [here](https://github.com/openshift/console/pull/3436#issuecomment-559194168)